lib/ES/Util.pm

```diff
@@ -85,9 +85,8 @@ sub build_chunked {
                 '-d' => 'book',
                 '-a' => 'showcomments=1',
                 '-a' => "lang=$lang",
-                '-a' => 'base_edit_url=' . $edit_url,
                 '-a' => 'root_dir=' . $root_dir,
-                $private ? ( '-a' => 'edit_url!' ) : (),
+                $private ? () : ( '-a' => "edit_url=$edit_url" ),
```

Comment:
I've flipped this around because just the presence of the edit_url attribute is enough for asciidoctor.